src/components/CardsWrapper/CardsWrapper.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import CardsWrapper from './CardsWrapper';
+
+describe('When rendered', () => {
+  it('should have cards wrapper', () => {
+    render(<CardsWrapper />);
+    expect(screen.getByText(new RegExp('cards', 'i'))).toBeTruthy();
```

Comment:
`getByRole('heading')` might be better alternative.
`CardsWrapper` renders list of cards and `getByText` might accidentally find completely different element.

Reply:
fix